mw-webapp/src/component/table/tableCell/TableCell.module.scss

```diff
@@ -0,0 +1,8 @@
+.cell {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  counter-reset: cnt;
```

Comment:
why do we need counter-reset? lets use simple ol tag

Reply:
Ol li doesnt work inside columns